### PR TITLE
add the contributor guide

### PR DESCRIPTION
**Pull request by ZhaiFeiyue**
# What does this PR do?

This PR is used to add the contributor guide which is linked to the  `CONTRIBUTING.md` of Huggingface transformers.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by ZhaiFeiyue** [comment]
HI @regisss, since we are enable and optimize models for optimum-habana continuously, and I notice that some general issues you have commented many times like code style, so I add a `Development` section in the Readme.md, in which shows how to contribute, to reduce your works.

File: README.md [hunk 1]
```diff
@@ -236,3 +236,8 @@ Please refer to Habana Gaudi's official [installation guide](https://docs.habana
 > Tests should be run in a Docker container based on Habana Docker images.
 >
 > The current version has been validated for SynapseAI 1.9.
+
+
+## Development
+
+Check the [contributor guide](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md) for instructions.
```
Comment (regisss) [inline README.md:243]:
```suggestion
Check the [contributor guide](https://github.com/huggingface/optimum/blob/main/CONTRIBUTING.md) for instructions.
```

**Review by regisss — COMMENTED** [review]
@ZhaiFeiyue You're fully right, let's add this to make it easier for newcomers to open and merge PRs :slightly_smiling_face: 

I just commented to link to the Optimum contribution guide because the Transformers one contains things that do not apply here (like `make quality` for example).

**Review by regisss — APPROVED** [review]
Thanks @ZhaiFeiyue :fire:

**Pull request merged**